include/etl/memory.h

```diff
@@ -1393,7 +1393,7 @@ namespace etl
     }
 
     //*********************************
-    void reset(pointer p_ = pointer()) ETL_NOEXCEPT
+    void reset(pointer p_) ETL_NOEXCEPT
```

Comment:
According to [cppreference](https://en.cppreference.com/w/cpp/memory/unique_ptr/reset), the signature for this member function is `void reset( pointer ptr = pointer() ) noexcept;`

Reply:
Right. I'll fix that later today